File: app/src/exoplayer/java/exoplayer/Utilities/UriUtil.java
package exoplayer.Utilities;

import android.content.Context;
import android.content.Intent;
import android.net.Uri;
import com.smile.karaokeplayer.Utilities.ContentUriAccessUtil;
import java.util.ArrayList;

public class UriUtil {
    private UriUtil() {};

    public static ArrayList<Uri> getUrisListFromIntent(Context context, Intent data) {
        return ContentUriAccessUtil.getUrisList(context, data);
    }
}

File: app/src/exoplayer/java/exoplayer/utilities/UriUtil.kt
package exoplayer.utilities

import android.content.Context
import android.content.Intent
import android.net.Uri
import java.util.ArrayList

object UriUtil {
    @JvmStatic
    fun getUrisListFromIntent(context: Context, data: Intent): ArrayList<Uri> {
        return ContentUriAccessUtil.getUrisList(context, data)
    }
}